core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.spec.ts

```diff
@@ -279,56 +324,62 @@ describe('DotContainerPropertiesComponent', () => {
             expect(field).toBeDefined();
         });
 
-        it('should button enable when max content greater then zero', fakeAsync(() => {
-            fixture.componentInstance.form.get('maxContentlets').setValue(2);
-            const contentTypeButton = de.query(By.css('[data-testId=showContentTypeAndCode]'));
+        it('should render content types when max-content greater then zero', fakeAsync(() => {
             spyOn(fixture.componentInstance, 'showContentTypeAndCode');
-            contentTypeButton.triggerEventHandler('click');
+            fixture.componentInstance.form.get('maxContentlets').setValue(0);
```

Comment:
Just to make sure if someone accidentally deletes or changes the condition this test fails.

```suggestion
            fixture.componentInstance.form.get('maxContentlets').setValue(0);
            tick();
            const codeEditorComponent = de.query(By.css('dot-container-code'));
            expect(codeEditoromponent).not.toBeDefined();
```

Reply:
Noted

Reply:
that's why I set a `maxContentlets` to 0 because after that condition is true. when it's loaded the first time it's rendering with content types.